CoreImportExportPlugins/src/au/gov/asd/tac/constellation/plugins/importexport/delimited/ImportController.java

```diff
@@ -170,7 +171,12 @@ public void setClearManuallyAdded(final boolean b) {
     }
 
     public void setDestination(final ImportDestination<?> destination) {
-        this.currentDestination = destination;
+        if (destination != null) {
```

Comment:
You could make this more concise using:
```
if (destination == null) {
    return;
}
this.currentDestination = destination;
```
...but is there an issue with setting `currentDestination` to null?

Reply:
No, I specifically am calling addDestination with 'null' from the RunPane.java call.  If it's already got a valid destination setup, I don't want to overwrite it.  I kept with the 'this.currentDestination' which should probably be just 'currentDestination' though.

Reply:
Sonar will flag the return statement in the void function so I'd say it would be ideal if it can be written without it (if there is a nice way of doing so)

Reply:
@CrucisGamma @antares1470  I've tagged you both on a different PR but I did a test to see if SQ complained about an empty return and it was OK with it. See #401.